cmd/ddev/cmd/get.go

```diff
@@ -45,454 +25,52 @@ ddev get --remove someowner/ddev-someaddonname,
 ddev get --remove ddev-someaddonname
 `,
 	Run: func(cmd *cobra.Command, args []string) {
-		officialOnly := true
-		verbose := false
-		bash := util.FindBashPath()
-		requestedVersion := ""
-
-		if cmd.Flags().Changed("version") {
-			requestedVersion = cmd.Flag("version").Value.String()
-		}
-
-		if cmd.Flags().Changed("verbose") {
-			verbose = true
+		// Create deprecation notice.
+		// Not we don't just set this as the "Deprecated" property of the cobra.Command
```

Comment:
Oh-so-trivial, add with [skip ci]
```suggestion
		// Note we don't just set this as the "Deprecated" property of the cobra.Command
```